docs/ja/reference/object/findKey.md

```diff
@@ -0,0 +1,31 @@
+# findKey
```

Comment:
We actually generate documents in an automated way; you don't have to manually create them.

Reply:
@raon0211 Oh, I see... I guess I won't have to write docs from next time.